Microsoft.Dotnet.Wpf.sln

```diff
@@ -1910,6 +1920,22 @@ Global
 		{AF9084C3-BF37-4A56-A851-89F3BAE731B3}.Release|x86.ActiveCfg = Release|Win32
 		{AF9084C3-BF37-4A56-A851-89F3BAE731B3}.Release|x86.Build.0 = Release|Win32
 		{AF9084C3-BF37-4A56-A851-89F3BAE731B3}.Release|x86.Deploy.0 = Release|Win32
+		{B2F2A89C-55C9-41B1-A645-0948609BD8BE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
@hughbe I think mapping these to the correct target platform (.sln x86 = .csproj x86, etc) should fix the error "There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference".